contribs/gmf/src/services/permalink.js

```diff
@@ -1169,4 +1169,62 @@ gmf.Permalink.prototype.createFilterGroup_ = function(prefix, paramKeys) {
 };
 
 
+/**
+ * Contains the layer name
+ * @param {!ol.Layer} layer The layer to inspect
+ * @param {string} name The layer name to find
+ * @return {boolean} The containing status
+ */
+gmf.Permalink.prototype.containsLayerName = function(layer, name) {
+  if (layer instanceof ol.layer.Group) {
+    for (const l of layer.getLayers().getArray()) {
+      if (this.containsLayerName(l, name)) {
+        return true;
+      }
+    }
+    return false;
+  } else {
+    return layer.get('layerNodeName') == name;
```

Comment:
It's working with WMTS layers, WMS mixed layers, etc ? (all cases ?)

Reply:
Just tested again, and effectively it's working for WMS and WMTS :-)